package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@apiaryio/metamorphoses",
-  "version": "0.13.7",
+  "version": "0.13.8",
```

Comment:
From semver

> Major version zero (0.y.z) is for initial development. Anything may change at any time. The public API should not be considered stable.

When less than v1, generally, people follow the rule of doing minor changes as patch versions and major changes as minor versions.